host/frontend/host-orchestrator/operator.go

```diff
@@ -0,0 +1,400 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+//     Unless required by applicable law or agreed to in writing, software
+//     distributed under the License is distributed on an "AS IS" BASIS,
+//     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+//     limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	pool := NewDevicePool()
+	polledSet := NewPolledSet()
+	var config interface{}
```

Comment:
Can this get its own type rather than using `interface{}`? The only requirement is that it eventually serializes into exactly the correct json, but that can be controlled with [annotations](https://pkg.go.dev/encoding/json#Marshal).

Reply:
Done